shardingsphere-infra/shardingsphere-infra-binder/src/main/java/org/apache/shardingsphere/infra/binder/statement/SQLStatementContext.java

```diff
@@ -18,14 +18,15 @@
 package org.apache.shardingsphere.infra.binder.statement;
 
 import org.apache.shardingsphere.infra.binder.segment.table.TablesContext;
+import org.apache.shardingsphere.infra.binder.type.SchemaAvailable;
 import org.apache.shardingsphere.sql.parser.sql.common.statement.SQLStatement;
 
 /**
  * SQL statement context.
  * 
  * @param <T> type of SQL statement
  */
-public interface SQLStatementContext<T extends SQLStatement> {
+public interface SQLStatementContext<T extends SQLStatement> extends SchemaAvailable {
```

Comment:
@cheese8 Inheriting the SchemaAvailable interface on SQLStatementContext will cause all StatementContext to be affected. A better way is to implement the SchemaAvailable interface on the DML statement Context.

Reply:
fixed